ci/environment.yml

```diff
@@ -26,3 +26,4 @@ clear_cache:
   when: manual
   script:
     - rm -rf .yarn node_modules packages/**/node_modules packages/*/lib packages/*/build packages/*/dist
+    - rm -rf .yarn-nix node_modules packages/**/node_modules packages/*/lib packages/*/build packages/*/dist
```

Comment:
Why not modify the first line? I mean `rm -rf .yarn .yarn-nix node_modules packages/**/node_modules packages/*/lib packages/*/build packages/*/dist` nope?

Reply:
Yes, it's better like this for sure. I, for some reason, thought that the rest is the path of the folders and files in that `.yarn `folder 👎